<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="DESKTOP-DJ4FDME" failures="5" tests="5" name="Test(failed)" time="0.134" errors="0" timestamp="2023-12-19T06:44:24 IST">
  <testcase classname="pkg2.FirstScenario3" name="first_testcase" time="0.03">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at pkg2.FirstScenario3.first_testcase(FirstScenario3.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="pkg2.FirstScenario" name="fifth_testcase" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at pkg2.FirstScenario.fifth_testcase(FirstScenario.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- fifth_testcase -->
  <testcase classname="pkg2.FirstScenario" name="first_testcase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at pkg2.FirstScenario.first_testcase(FirstScenario.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="pkg2.FirstScenario7" name="first_testcase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at pkg2.FirstScenario7.first_testcase(FirstScenario7.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="pkg2.FirstScenario4" name="first_testcase" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [Hello1] but found [Hello]">
      <![CDATA[java.lang.AssertionError: expected [Hello1] but found [Hello]
at pkg2.FirstScenario4.first_testcase(FirstScenario4.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- first_testcase -->
  <testcase classname="pkg2.FirstScenario" name="forth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- forth_testcase -->
  <testcase classname="pkg2.FirstScenario" name="sixth_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- sixth_testcase -->
  <testcase classname="pkg2.FirstScenario" name="third_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- third_testcase -->
  <testcase classname="pkg2.FirstScenario" name="second_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- second_testcase -->
  <testcase classname="pkg2.FirstScenario" name="seventh_testcase" time="0.0">
    <ignored/>
  </testcase> <!-- seventh_testcase -->
</testsuite> <!-- Test(failed) -->
